k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl

```diff
@@ -118,12 +122,14 @@ cortx:
     confd:
       endpoints: {{- toYaml .Values.cortxMotr.confdEndpoints | nindent 6 }}
     clients:
+    {{- if .Values.cortxRgw.enabled }}
       - name: rgw
         num_instances: 1  # number of instances *per-pod*
         endpoints: {{- toYaml .Values.cortxMotr.rgwEndpoints | nindent 8 }}
+    {{- end }}
     {{- if gt (len .Values.cortxMotr.clientEndpoints) 0 }}
       - name: motr_client
-        num_instances: {{ len .Values.cortxMotr.clientEndpoints }}
+        num_instances: {{ .Values.cortxMotr.clientInstanceCount | int }}
```

Comment:
I believe this is supposed to be the number of instances _per-pod_, not the number of total endpoints. The endpoints value reflects the number of nodes instead.